Aufgabe 3 Blatt 1

LukasNebelung opened the pull request:
Habe die Lösungen für die Aufgabe in die Kommentare geschrieben und die geforderten Änderungen in den Code geschrieben.
Die letzte Zeile von Till hat nicht funktioniert. Diese Version klapp jetzt. 
Die Teilaufgabe b habe ich zwar gelöst, kann aber nicht wirklich erklären, warum sie funktioniert.

LukasNebelung committed:
Aufgabe 3 Blatt 1

Habe die Lösungen für die Aufgabe in die Kommentare geschrieben und die geforderten Änderungen in den COde geschrieben.
Die letzte Zeile von Till hat nicht funktioniert. Diese Version klapp jetzt. 
Die Teilaufgabe b habe ich zwar gelöst, kann aber nicht wirklich erklären, warum sie funktioniert.

TillSebastian commented:
ich hab die aufgabe so interpretiert dass man gerade kein == schreiben sollte wenn dann wieder ein = false damit ein false rauskommt

TillSebastian commented:
aber ansonsten würde ich es so abgeben

LukasNebelung committed:
Update drei.java

Hab jetzt mal beides eingefügt. Dann kann sich der Tutor das was gemeint war raussuchen

LukasNebelung commented:
hab das jetzt nochmal geändert und beide Zeilen eingefügt. Dann kann der Tutor es sich aussuchen :)

TillSebastian committed:
Update drei.java

nur um noch auf nummer sicher zu gehen :)

LukasNebelung commented:
Wir können ja noch die Aufgabenstellung reinkopieren. Dann kann  er uns gar nix XD
